cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/BrokerCapacityConfigFileResolver.java

```diff
@@ -156,12 +203,15 @@ private static boolean isJBOD(Map<Resource, Object> brokerCapacity) {
             totalDiskCapacity += Double.parseDouble(diskEntry.getValue());
           }
           totalCapacity.put(resource, totalDiskCapacity);
+        } else if (hasNumCores && resource == Resource.CPU) {
+          totalCapacity.put(resource, DEFAULT_CPU_CAPACITY_WITH_CORES);
```

Comment:
(This maybe correct, just to confirm) so in this patch, if `num.cores` is specified, the cpu capacity is `100` instead of `100 * num_core`.

Reply:
Yes, this is the intended behavior.